Rewrite README file in markdown (#97)
sensu/sensu-go-ansible: docs/source/index.rst

Sensu Go Ansible Collection
===========================

Sensu Go `Ansible Collection`_ contains:

* Ansible role to install `Sensu Go`_ on your nodes.
* Ansible modules for interacting with `Sensu Go`_ REST API.

.. _Ansible Collection:
   https://docs.ansible.com/ansible/latest/user_guide/collections_using.html
.. _Sensu Go: https://docs.sensu.io/sensu-go/latest

Installation
------------

If we would like to install the Sensu Go collection, we need to have Ansible
2.9 or later installed. Once we have taken care of this, we can install the
Sensu Go collection from `Ansible Galaxy`_ by running::

   $ ansible-galaxy collection install sensu.sensu_go

.. _Ansible Galaxy: galaxy.ansible.com

Usage
-----

> You can use the namespaced modules::

Reviewer: ```suggestion
We can use the namespaced modules::
```

   - hosts: localhost
     tasks:
       - name: List Assets
         sensu.sensu_go.asset_info:
         register: assets

       - name: List Filters
         sensu.sensu_go.filter_info:
         register: filters

> Or add the collection to the search path and use bare names::

Reviewer: ```suggestion
Or use the `collections` keyword to create the search path, then use bare names:
```

Sensu Go Ansible Collection
===========================

Sensu Go `Ansible Collection`_ contains:

> * Ansible role to install `Sensu Go`_ on your nodes.

Reviewer 2: Roles ;)

Author: I already hate the duplication ;)
